universal-application-tool-0.0.1/app/services/applicant/Block.java

```diff
@@ -66,6 +66,20 @@ public String getDescription() {
     return blockDefinition.description();
   }
 
+  /** This block is an enumerator block if its {@link BlockDefinition} is an enumerator. */
+  public boolean isEnumerator() {
```

Comment:
question: is there a rename in the future to standardize "repeater"/"enumerator"?

Reply:
yes! #919

Reply:
Awesome!